OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -165,62 +513,345 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <desc>Poor handwriting. Unexperienced scribe who tends to scribal
+                                    errors and to a rushed script.</desc>
+                            </handNote>
+
                         </handDesc>
-                        
-                       <!-- <decoDesc>
-                         
-                        </decoDesc>-->
-                        
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
+                        <!--
+                                     <decoDesc> Ornamental bands are drawn on the incipit page of <ref
+                                target="#ms_i1"/> and of all its sections (<locus
+                                target="#4r #15v #27r #36r #45v #55v #66v"/>) except for section
+                                <ref target="#ms_i1.8"/> for which the space was left unfilled
+                                (<locus target="#74r"/>). The scribe left a space for an ornamental
+                            band also on the incipit page of <ref target="#ms_2 #ms_3"/> (<locus
+                                target="#80v #95v"/>). </decoDesc>-->
+
+                        <additions>
+                            <list>
+                                <item xml:id="a1" corresp="#h2">
+                                    <locus from="119r" to="120v"/>
+                                    <desc type="GuestText">
+                                        <title ref="LIT3308Nicene" type="complete"/>
+                                    </desc>
+                                    <q xml:lang="gez">
+                                        <locus target="#119r"/>
+                                        <seg part="I"> በአሀዱ፡ አምላክ፡ እግዚአብሔር፡ አብ፡ አኀዜ፡ ኵሉ፡ ገባሬ፡ ሰማያት፡
+                                            ወምድር፡ ዘያስተርኢ፡ ወዘኢያስተርኢ። </seg>
+                                        <locus target="#120r"/><seg part="F"> ወነአምን፡ በአ<add
+                                                place="above">ሐ</add>ቲ፡ ጥምቀት። ለሥርየተ፡ ኃጢ<add
+                                                place="above">አ</add>ት ፡ ወን<pb n="120v"/>ሴፎ፡ ትንሣኤ፡
+                                            ሙታን። ወሕይወት፡ ዘይመጽዕ፡ ለዓለመ፡ ዓለም። </seg>
+                                    </q>
+                                </item>
+                                <item xml:id="a2" corresp="#h2">
+                                    <locus target="120v"/>
+                                    <desc type="GuestText">
+                                        <title ref="LIT5991PraiseGod" type="incomplete"/>The text
+                                        stops in the middle of a word on the bottom of <locus
```

Comment:
Here we could start to think of how we would like this to be printed. There will be the title of LIT5991PraiseGod (probably taken from the work record), and then the text, so there might be a stop/comma/space between them.

Reply:
Title + colon + text ?